The dependency on ini4j is changed to type 'system', which is an ugly workaround
to use a jar file not in Debian's local maven repository. Other than updating the ini4j
package to add the maven parts, there seems to be no other way around this.
--- a/pom.xml
+++ b/pom.xml
@@ -55,6 +55,8 @@
         <dependency>
             <groupId>net.sourceforge</groupId>
             <artifactId>ini4j</artifactId>
+	    <scope>system</scope>
+	    <systemPath>/usr/share/java/ini4j.jar</systemPath>
         </dependency>
         <dependency>
             <groupId>emi</groupId>
